autoupgrade.php

```diff
@@ -184,7 +184,10 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
     {
         require_once _PS_ROOT_DIR_ . '/modules/autoupgrade/classes/UpgradeTools/Translator.php';
 
-        $translator = new \PrestaShop\Module\AutoUpgrade\UpgradeTools\Translator();
+        $translator = new \PrestaShop\Module\AutoUpgrade\UpgradeTools\Translator(
+            _PS_ROOT_DIR_ . '/modules/autoupgrade/translations/',
```

Comment:
```suggestion
            _PS_ROOT_DIR_ . DIRECTORY_SEPARATOR . 'modules' . DIRECTORY_SEPARATOR . 'autoupgrade' . DIRECTORY_SEPARATOR . 'translations' . DIRECTORY_SEPARATOR,
```
you sue DIRECTORY_SEPARATOR in `classes/UpgradeContainer.php` but not her. I think it's a good thing to use it.